packages/nimble-tokens/dist/fonts/css/fonts.css

```diff
@@ -14,6 +21,14 @@
   font-display: swap;
 }
 
+@font-face {
+  font-family: "Source Sans Pro light-fallback";
+  font-weight: 300;
+  size-adjust: 93.75%;
+  ascent-override: 110%;
+  src: local("Arial");
```

Comment:
Can you make some documentation for how to make a new Fallback font so if fonts change or new ones are added we can reproduce these values?

I would say pull the readme in /nimble-tokens/dist/fonts/assets/README.md up to the /nimble-tokens/dist/fonts directory and add the discussion there.

Reply:
Done